app.js

```diff
@@ -1,8 +1,10 @@
 const express = require ('express')
-const {fetchallCategories} = require()
+const {fetchallCategories} = require("./controllers/categoryControllers")
 
 const app = express()
 
+app.get("/api/categories", fetchallCategories)
 
+module.exports = app
 
```

Comment:
An error handler for any 500 errors could be useful at this stage to tie in with controller catch block.